gen/dcos-config.yaml

```diff
@@ -1350,6 +1350,22 @@ package:
           "registrar",
           "allocator",
        ]
+      # Reads 'mntr' stats from one or many zookeeper servers
+      [[inputs.zookeeper]]
+        ## An array of address to gather stats about. Specify an ip or hostname
+        ## with port. ie localhost:2181, 10.0.0.1:2181, etc.
+        ## If no servers are specified, then localhost is used as the host.
```

Comment:
Maybe use "If no host is specified," here, to match port line below and also to make it clear that the value used `":2181"` represents `"localhost:2181"`.